e2e/tests/dataPanel.spec.ts

```
@@ -0,0 +1,191 @@
+    test(`@readwrite Validate delete node via API and validation via UI`, async () => {
+        const graphName = getRandomString('datapanel');        
+        await apicalls.addGraph(graphName);
+        await apicalls.runQuery(graphName, 'CREATE (a:Person {name: "Alice", age: 30}), (b:Person {name: "Bob"})');
+        await apicalls.runQuery(graphName, 'MATCH (b:Person {name: "Alice"}) DELETE b');
+        const graph = await browser.createNewPage(DataPanel, urls.graphUrl);
+        await browser.setPageToFullScreen();
+        await graph.selectExistingGraph(graphName);
+        await graph.insertQuery(FETCH_FIRST_TEN_NODES);
+        await graph.clickRunQuery();
+        const nodes = await graph.getNodesGraphStats();
+        expect(parseInt(nodes ?? "")).toBe(1);
```

ESLint

[error] Missing radix parameter.

(radix)